src/App.js

```diff
@@ -1,13 +1,58 @@
 import React from 'react';
+import TodoForm from './components/TodoComponents/TodoForm'
+import TodoList from './components/TodoComponents/TodoList'
+import './app.css'
 
+
+const todos = []
 class App extends React.Component {
+  constructor () {
+    super();
+    this.state = {
+      todos: todos,
+      task: "", 
+      
+    }
+  }
+
+  addTask = e => {
```

Comment:
```suggestion
    addTask = e => {
    e.preventDefault();
      if (this.state.task === '' ) {
    return null
  }
    
    const newTask = {
      task: this.state.task,
      id: Date.now(),
      completed: false,
    }

    this.setState({
      todos: [...this.state.todos, newTask],
      task: "",
    });

  };
```

So if you see that if conditional I added at the top, what's cool about this is it's basically saying if you try to add a todo with no value inside your input then it won't add anything.